app/lib/one_sky.rb

```diff
@@ -10,15 +10,21 @@ module OneSky
   def self.download_translated_phrases(filename, project_id:, language_code:)
     logger.info "Downloading translated phrases for: #{filename} with language: #{language_code}"
 
-    response = RestClient.get "https://platform.api.onesky.io/1/projects/#{project_id}/translations",
-      params: headers(language_code).merge(
-        source_file_name: filename, export_file_name: filename
-      )
+    no_onesky_project = false
+    begin
+      response = RestClient.get "https://platform.api.onesky.io/1/projects/#{project_id}/translations",
+        params: headers(language_code).merge(
+          source_file_name: filename, export_file_name: filename
+        )
+    rescue RestClient::BadRequest
+      no_onesky_project = true
+    end
 
-    if response.code == 204
+    if no_onesky_project || response.code == 204
       logger.info "No translated phrases found for: #{filename} with language: #{language_code}"
       return {}
     end
+
```

Comment:
I'm intrigued by this logic, was this only for testing locally?

I do want to improve the onesky translation behavior for download translations, but I haven't fully documented how I would actually want it to behave.

The behavior should throw an error if something references a translation that doesn't actually exist, but not require OneSky if there are no actual translations.

Reply:
It came out of some of my testing, I made a draft of a new language that had no data with it, and it was giving "bad request".  It was hindering my testing so I added this.  I didn't think there were any translations in that language because I had just added a new draft, but maybe there was?  To be safe maybe we should revert this before completion.

Reply:
I still need to digest this change and the impact it will have. This is mostly a note for myself and should block merging this PR until I have a chance to look into this.

Reply:
I'm good with this change.